GVRf/Framework/framework/src/main/java/org/gearvrf/script/GVRScriptManager.java

```diff
@@ -217,19 +243,17 @@ public GVRScriptFile getScriptFile(IScriptable target) {
      * @throws GVRScriptException if script processing error occurs.
      */
     public GVRScriptFile loadScript(GVRAndroidResource resource, String language) throws IOException, GVRScriptException {
-        if (getEngine(language) == null) {
-            mGvrContext.logError("Script language " + language + " unsupported", this);
-            throw new GVRScriptException(String.format("The language is unknown: %s", language));
-        }
+        //final Object[] getV8Engine = new Object[0];
 
         GVRScriptFile script = null;
-        if (language.equals(LANG_LUA)) {
-            script = new GVRLuaScriptFile(mGvrContext, resource.getStream());
-        } else if (language.equals(LANG_JAVASCRIPT)) {
-            script = new GVRJavascriptScriptFile(mGvrContext, resource.getStream());
+        try {
+            InputStream inputStream = resource.getStream();
```

Comment:
Precautionary you should use a try-with-resources statement. See https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html